packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerCache.java

```diff
@@ -34,42 +33,48 @@
       "flutter_image_picker_pending_image_uri";
   private static final String SHARED_PREFERENCES_NAME = "flutter_image_picker_shared_preference";
 
-  private static SharedPreferences getFilePref;
+  private SharedPreferences prefs;
 
-  static void setUpWithActivity(Activity activity) {
-    getFilePref =
-        activity
-            .getApplicationContext()
-            .getSharedPreferences(SHARED_PREFERENCES_NAME, Context.MODE_PRIVATE);
+  ImagePickerCache(Context context) {
+    prefs = context.getSharedPreferences(SHARED_PREFERENCES_NAME, Context.MODE_PRIVATE);
   }
 
-  static void saveTypeWithMethodCallName(String methodCallName) {
+  void saveTypeWithMethodCallName(String methodCallName) {
     if (methodCallName.equals(ImagePickerPlugin.METHOD_CALL_IMAGE)) {
       setType("image");
     } else if (methodCallName.equals(ImagePickerPlugin.METHOD_CALL_VIDEO)) {
       setType("video");
     }
   }
 
-  private static void setType(String type) {
-    if (getFilePref == null) {
-      return;
-    }
-    getFilePref.edit().putString(SHARED_PREFERENCE_TYPE_KEY, type).apply();
+  private void setType(String type) {
+
+    prefs.edit().putString(SHARED_PREFERENCE_TYPE_KEY, type).apply();
   }
 
-  static void saveDemensionWithMethodCall(MethodCall methodCall) {
-    Double maxWidth = methodCall.argument("maxWidth");
-    Double maxHeight = methodCall.argument("maxHeight");
+  void saveDimensionWithMethodCall(MethodCall methodCall) {
+    Double maxWidth = methodCall.argument(MAP_KEY_MAX_WIDTH);
+    Double maxHeight = methodCall.argument(MAP_KEY_MAX_HEIGHT);
     setMaxDimension(maxWidth, maxHeight);
   }
 
-  private static void setMaxDimension(Double maxWidth, Double maxHeight) {
-    if (getFilePref == null) {
-      return;
-    }
+  Map<String, Double> getMaxDimensions() {
```

Comment:
where is this method used?

Reply:
I forgot to remove it from my first PR. Removed.